src/pages/ApplyVerify.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from 'react';
+
+import BlockButton from '@/components/common/button/BlockButton';
+import LabelButton from '@/components/common/button/LabelButton';
+import Icon from '@/components/common/icon/Icon';
+import InputField from '@/components/common/input/InputField';
+import Label from '@/components/common/label/Label';
+import ProgressIndicator from '@/components/common/progress/ProgressIndicator';
+import Title from '@/components/common/title/Title';
+import { APPLY_TITLE } from '@/constants/applyPageData';
+
+function ApplyVerify() {
+  const [isReady, setIsReady] = useState(false);
+  const [isExist, setIsExist] = useState(false);
```

Comment:
```suggestion
  const [isNew, setIsNew] = useState(false);
```

```suggestion
  const [isNewApplicant, setIsNewApplicant] = useState(false);
```

은 어떠신가욥?! `isExist`라는 네이밍은 다소 모호하게 느껴져서 제안 드려요!

Reply:
[3adca65](https://github.com/JECT-Study/JECT-Official-WebSite-Client/pull/82/commits/3adca658e22fdaa0d784f000a70c6f819883f337) isNewApplicant 좋네요!! 요걸로 수정했습니다! 👍